.travis/after_success_storybook.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
There is another tasks than storybooks, you could talk about "after_success_demo.sh"